job-service-container/src/test/java/com/hpe/caf/services/job/api/JobServiceIT.java

```diff
@@ -1249,16 +1250,16 @@ private int getRowsInDeleteLog(java.sql.Connection dbConnection) throws SQLExcep
      *         {@link AssertionError} on timeout.
      * @throws Exception
      */
-    private static Supplier<TaskMessage> getMessageFromQueue(final QueueManager queueManager)
+    private static Supplier<QueueTaskMessage> getMessageFromQueue(final QueueManager queueManager)
         throws Exception
     {
         final ExecutionContext context = new ExecutionContext(false);
         context.initializeContext();
         final Timer timer = getTimer(context);
 
-        final List<TaskMessage> result = new ArrayList<>();
+        final List<QueueTaskMessage> result = new ArrayList<>();
         queueManager.start(message -> {
-            result.add(message);
+            result.add((QueueTaskMessage)message);
```

Comment:
This line is alerting me to a problem.  You shouldn't need a cast.  Why is one required?

Reply:
Removed after updating the workerframework.test